scripts/scripts/enrich_sentences.py

```
@@ -0,0 +1,223 @@
+
+from library.gemini_client import get_global_gemini_client
+
+logging.basicConfig(
+    level=logging.INFO,
+    format="%(asctime)s - %(name)s - %(levelname)s - %(message)s",
+    handlers=[logging.StreamHandler(sys.stderr)],
+)
+logger = logging.getLogger(__name__)
+
+
+class DifficultyRating(BaseModel):
```

Pylint (3.3.7)

[convention] Missing class docstring

(C0115)